ci/docker-entrypoint.sh

```diff
@@ -2,10 +2,12 @@
 set -e
 
 function start_dgraph() {
+    dgraph version
```

Comment:
Do we need to print the version?

Reply:
Its nice to have information in the logging